website/docs/d/nsxv_firewall_rule.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxv_firewall_rule"
+sidebar_current: "docs-vcd-data-source-nsxv-firewall-rule"
+description: |-
+  Provides a vCloud Director firewall rule data source for advanced edge gateways (NSX-V). This can
+  be used to read existing rules by ID and use its attributes in other resources.
+---
+
+# vcd\_nsxv\_firewall\_rule
+
+Provides a vCloud Director firewall rule data source for advanced edge gateways (NSX-V). This can be
+used to read existing rules by ID and use its attributes in other resources.
+
+~> **Note:** This data source requires advanced edge gateway. For non-advanced edge gateways please
+use the [`vcd_firewall_rules`](/docs/providers/vcd/r/firewall_rules.html) resource.
+
+## Example Usage
+
+```hcl
+data "vcd_nsxv_firewall_rule" "my-rule" {
+  org                 = "my-org"
+  vdc                 = "my-org-vdc"
+  edge_gateway        = "my-edge-gw"
+
+  rule_id = "133048"
```

Comment:
This is UI rule or real Id ?:)

Reply:
This is rule id

Reply:
So this is the "real" ID. I actually tried to make distinction that ID is the real ID, while UI number means that number. There is also reference about it.

Reply:
so maybe this would help:
```suggestion
 # real rule Id, not UI Id
  rule_id = "133048"
```

Reply:
Added